plugins.txt

```diff
@@ -1,4 +1,4 @@
-blueocean:1.8.2
-workflow-support:incrementals;org.jenkins-ci.plugins.workflow;2.21-rc591.43d37d4d080a
```

Comment:
I think we need to align/use the same version everywhere to avoid surprises.
Not the same as we decided to use on Evergreen side. Or should it be updated there?

https://github.com/jenkins-infra/evergreen/blob/e52b6c4efbf0cbcf27343280d094f08eabe5e7cb/services/essentials.yaml#L38